src/System.Memory/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Float.cs

```diff
@@ -76,10 +76,6 @@ public static bool TryFormat(float value, Span<byte> destination, out int bytesW
             {
                 case 'g':
                 case 'G':
-                    if (format.Precision != StandardFormat.NoPrecision)
-                        throw new NotSupportedException(SR.Argument_GWithPrecisionNotSupported);
-                    break;
```

Comment:
There's a similar check for decimal:
https://github.com/dotnet/corefx/blob/b86783ef9525f22b0576278939264897464f9bbd/src/Common/src/CoreLib/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Decimal.cs#L44
Is that one ok, or should it be removed as well?

Reply:
That check still exists in .NET Core 3.0 as well.

I don't believe it is required for `decimal` however, as I believe it produces a round-trippable result by default.